wsmaster/che-core-api-factory-github-common/src/main/java/org/eclipse/che/api/factory/server/github/AbstractGithubURLParser.java

```diff
@@ -78,6 +79,7 @@ public abstract class AbstractGithubURLParser {
     this.apiClient = githubApiClient;
     this.disableSubdomainIsolation = disableSubdomainIsolation;
     this.providerName = providerName;
+    this.isGitHubServer = !isNullOrEmpty(oauthEndpoint);
```

Comment:
```suggestion
    this.isGitHubServer = !isNullOrEmpty(oauthEndpoint) || GITHUB_SAAS_ENDPOINT.equals(oauthEndpoint);
```
This way?

Reply:
Thank you for the suggestion, but I have commited: `&& !GITHUB_SAAS_ENDPOINT.equals(oauthEndpoint)`